templates/_reservations.html

```diff
@@ -5,17 +5,17 @@
 
   <div class="modal-dialog">
     <div class="modal-content left">
-        <form method="get" id="TU_form" target="_blank" action="https://www.opentable.com/restaurant-search.aspx">
```

Comment:
already deleted both the id and the hardcoded id from the html, can check and merge if everything is okay.